modules/api/test/TestPlan.md

```diff
@@ -33,10 +33,12 @@ See specification for the expected response format.
 
 
 #### Candidate creation - POST /candidates
-Candidate creation
-This is done by sending a POST request to /candidates with the required data. Note the usage of the Authorization header.
+Candidate creation is done by sending a POST request to /candidates with the required data. Note the usage of the Authorization header. For a candidate to be created, the candidate's information must also be included. The following command contains the minimal information necessary for the request to create a new candidate (see [docs](https://github.com/aces/Loris/blob/23.0-release/docs/API/LorisRESTAPI_v0.0.3.md#30-candidate-api) for format instructions):
```

Comment:
This should be a relative URL, not an absolute URL.

Reply:
@laemtl  - a good note for the documentation guide? :) 
p.s. could you help review Simon's last change on this today? merci